lib/github/search.rb

```diff
@@ -7,7 +7,7 @@ class Search
     def initialize(access_token, options = { auto_paginate: false })
       options[:access_token] = access_token
 
-      @search_client = Octokit::Client.new(options)
+      @search_client = GitHubClassroom.github_client(options)
```

Comment:
Yeah, check out this method. auto_paginate is passed in through options and if we do a reassignment of that within GitHubClassroom.github_client, we're changing behaviour inadvertently. I'll push one more commit that makes it so the factory for the client doesn't make a decision on pagination. It defers to whatever the caller passes in.

Reply:
The change to the factory for this comment ☝️ was added... however there was a small side-effect: existing consumers of the factory method had their `auto_paginate` behavior changed (which previously defaulted to true). 

These are the three impacted call sites:

https://github.com/mdwheele/classroom/blob/281877f6a0d636fe1413d7f69deaf102956dc8f9/app/models/github_model.rb#L181
https://github.com/mdwheele/classroom/blob/281877f6a0d636fe1413d7f69deaf102956dc8f9/app/models/github_user.rb#L16
https://github.com/mdwheele/classroom/blob/281877f6a0d636fe1413d7f69deaf102956dc8f9/lib/github/token.rb#L46

Reply:
I think all of these are fixed now! 🎉